apps/namadillo/src/App/Ibc/IbcTransfer.tsx

```diff
@@ -35,6 +35,29 @@ export const IbcTransfer: React.FC = () => {
   const performIbcTransfer = useAtomValue(ibcTransferAtom);
   const defaultAccounts = useAtomValue(allDefaultAccountsAtom);
 
+  const transactionFee = useMemo(() => {
+    if (typeof registry !== "undefined") {
+      // TODO: can we get a better type for registry to avoid optional chaining?
+      // TODO: some chains support multiple fee tokens - what should we do?
```

Comment:
Good point - maybe at some point we could let user's choose a fee token that has balance, otherwise prevent them from submitting? I think that can be done later! Defaulting to native token is probably a good idea for now

Reply:
I've added #1203 to track this.